plugins/GSdx/res/tfx.fx

```diff
@@ -575,6 +589,12 @@ float4 ps_color(PS_INPUT input)
 		c.rgb *= c.rgb < 128./255;
 	}
 
+#if PS_CHANNEL_FETCH == 1
+	float4 t = fetch_blue(uv);
```

Comment:
Seems like you use an undefined uv. This should be ` input.uv`. However, you don't need uv here. ;)